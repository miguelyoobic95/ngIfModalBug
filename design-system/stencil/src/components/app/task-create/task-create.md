---
name: TaskCreate
category: Core Components
---

```task-create.html
    <task-create></task-create>
```

## Attributes

|Name|Type|Default|Description|
|---|---|---|---|
|`text`|string|   |the text inside the label|

## CSS

|Type|Name|Description|
|---|---|---|
|Accent|`accent`|task-create accent|
|Dark|`dark`|task-create dark|
|Success|`success`|task-create success|
|Danger|`danger`|task-create danger|
|Info|`info`|task-create info|
|Warning|`warning`|task-create warning|
|Gradient Accent|`gradient-accent`|task-create gradient-accent|
|Gradient Dark|`gradient-dark`|task-create gradient-grey|
|Gradient Success|`gradient-success`|task-create gradient-success|
|Gradient Danger|`gradient-danger`|task-create gradient-danger|
|Gradient Info|`gradient-info`|task-create gradient-info|
|Gradient Warning|`gradient-warning`|task-create gradient-warning|

## Events
|Attr|Description|
|---|---|---|
|`yooXXXX`|small label|