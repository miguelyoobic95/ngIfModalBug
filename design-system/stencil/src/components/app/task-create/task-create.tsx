import { Component, Element } from '@stencil/core';

@Component({
    tag: 'yoo-task-create',
    styleUrl: 'task-create.scss',
    scoped: true
})
export class YooTaskCreateComponent {

     @Element() host: HTMLStencilElement;

    render(): JSX.Element {
        return (
            <div>TaskCreate needs a proper template</div>
        );
    }
}
