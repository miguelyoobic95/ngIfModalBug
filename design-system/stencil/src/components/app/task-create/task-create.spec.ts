import { TestWindow } from '@stencil/core/testing';
import { YooTaskCreateComponent } from './task-create';

describe('TaskCreateComponent', () => {
    it('should build', () => {
        expect(new YooTaskCreateComponent()).toBeTruthy();
    });

    describe('Rendering', () => {
        let element, window;
        beforeEach( async () => {
            window = new TestWindow();
            element = await window.load({
                components: [YooTaskCreateComponent],
                html: '<yoo-task-create></yoo-task-create>'
            });
        });

        it ('Should render', async () => {
            expect(element).toMatchSnapshot();
        });
    });

    describe('Events', () => {

        let emitEvent, window;
        beforeEach( async () => {
            window = new TestWindow();
            emitEvent = jest.fn();
        });
    });
});