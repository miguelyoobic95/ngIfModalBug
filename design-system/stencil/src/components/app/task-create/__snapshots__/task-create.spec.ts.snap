// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskCreateComponent Rendering Should render 1`] = `
<yoo-task-create
  class="hydrated"
  data-yoo-task-create-host=""
>
  
  <div
    data-yoo-task-create=""
  >
    TaskCreate needs a proper template
  </div>
</yoo-task-create>
`;
